[rocprofiler-compute] Update Documentation for Strix Halo (#4794)
ROCm/rocm-systems: projects/rocprofiler-compute/docs/conceptual/rdna/gl2-cache.rst

.. meta::

>    :description: ROCm Compute Profiler — RDNA3.5 GL2C and GCEA / DRAM interface metrics

Reviewer: ```suggestion
   :description: Learn about GL2C metrics in ROCm Compute Profiler, including the GL1C–GL2 interface, cache performance, bandwidth, GCEA, and DRAM interfaces on RDNA 3.5 (gfx1151).
```

   :keywords: ROCm Compute Profiler, RDNA, gfx1151, GL2C, GL2, GCEA

.. _rdna-gl2:

===============================
GL2 cache (GL2C) and GCEA
===============================

> On gfx1151, the last-level on-chip cache seen by most clients is GL2C (RDNA

Reviewer: ```suggestion
On gfx1151, GL2C (RDNA naming for what Instinct documentation refers to as L2/TCC) is the last-level on-chip cache for most clients.
```

> naming for what Instinct documentation calls L2 / TCC). Traffic that leaves GL2

Reviewer: ```suggestion
 Traffic leaving GL2
```

heads toward GCEA and DRAM through the DRAM read/write, SARB, and
return interfaces in the panel YAMLs.

> For Instinct L2 (TCC) coherence, channel hashing, and fabric metrics, see

Reviewer 2: Consistency note here- can we match what the other pages state like "For CDNA arch on the MI Instinct products see" (paraphrasing but can be copied from another page) instead of just saying "Instinct"

> For Instinct L2 (TCC) coherence, channel hashing, and fabric metrics, see

Reviewer: ```suggestion
For Instinct L2 (TCC) coherence, channel hashing, and fabric metrics on CDNA architecture across MI-series GPUs, see
```

> :doc:`../cdna/l2-cache`.

Reviewer: ```suggestion
:doc:`../cdna/l2-cache` under CDNA-CDNA4.
```

GL1C to GL2C and GL2C panels
=============================

> 

Reviewer: An introductory description for GL1C to GL2C and GL2C panels is recommended.

Reviewer 3: will be worked later with separate PR

> GL1C GL1C-GL2 Interface (panel)

Reviewer: ```suggestion
GL1C GL1C-GL2 interface (panel)
```

Reviewer: Is GL1C repeated twice here?

---------------------------------

.. tab-set::

   .. tab-item:: RDNA 3.5 (gfx1151)
      :selected:

      .. jinja:: rdna1151-gl1c-gl1c-gl2-interface-gfx1151
         :file: _templates/metrics_table.j2

> GL2C Cache Performance

Reviewer: ```suggestion
GL2C cache performance
```

----------------------

.. tab-set::

   .. tab-item:: RDNA 3.5 (gfx1151)
      :selected:

      .. jinja:: rdna1151-gl2c-cache-performance-gfx1151
         :file: _templates/metrics_table.j2

> GL2C Request Statistics

Reviewer: ```suggestion
GL2C request statistics
```

-----------------------

.. tab-set::

   .. tab-item:: RDNA 3.5 (gfx1151)
      :selected:

      .. jinja:: rdna1151-gl2c-request-statistics-gfx1151
         :file: _templates/metrics_table.j2

> GL2C Bandwidth

Reviewer: ```suggestion
GL2C bandwidth
```

--------------

.. tab-set::

   .. tab-item:: RDNA 3.5 (gfx1151)
      :selected:

      .. jinja:: rdna1151-gl2c-bandwidth-gfx1151
         :file: _templates/metrics_table.j2

GCEA and DRAM interfaces
=========================

> 

Reviewer: An introductory description for GCEA and DRAM interfaces is recommended.

Reviewer 3: will be worked later with separate PR

> DRAM Read Interface

Reviewer: ```suggestion
DRAM read interface
```

-------------------

.. tab-set::

   .. tab-item:: RDNA 3.5 (gfx1151)
      :selected:

      .. jinja:: rdna1151-dram-read-interface-gfx1151
         :file: _templates/metrics_table.j2

> DRAM Write Interface

Reviewer: ```suggestion
DRAM write interface
```

--------------------

.. tab-set::

   .. tab-item:: RDNA 3.5 (gfx1151)
      :selected:

      .. jinja:: rdna1151-dram-write-interface-gfx1151
         :file: _templates/metrics_table.j2

System Arbiter (SARB)
---------------------

.. tab-set::

   .. tab-item:: RDNA 3.5 (gfx1151)
      :selected:

      .. jinja:: rdna1151-system-arbiter-sarb-gfx1151
         :file: _templates/metrics_table.j2

> Return Interface

Reviewer: ```suggestion
Return interface
```
